packages/scandipwa/src/query/Cart.query.js

```diff
@@ -149,12 +156,37 @@ export class CartQuery {
             'shipping_incl_tax',
             'shipping_tax_amount',
             'shipping_method',
+            ...checkoutData,
             'is_in_store_pickup_available',
             this._getCartItemsField(),
             this._getAppliedTaxesField()
         ];
     }
 
+    _getOrderShippingAddressField() {
+        return new Field('shipping_address')
+            .addFieldList(this._getOrderAddressFields());
+    }
+
+    _getOrderAddressFields() {
+        return [
+            'city',
+            'country_id',
+            'firstname',
+            'lastname',
+            'postcode',
+            'region',
+            'telephone',
+            'vat_id',
+            'email',
+            this._getOrderAddressStreetField()
+        ];
+    }
+
+    _getOrderAddressStreetField() {
+        return new Field('street');
+    }
```

Comment:
There is no reason to return just a single field.
Please put it as the string to the `_getOrderAddressFields` method.